spec/reek/smells/attribute_spec.rb

```diff
@@ -47,26 +35,10 @@ class Klass
       ').to_not reek_of(:Attribute)
     end
 
-    it 'records attr attribute in a module' do
+    it 'records attr_writer attribute in a module' do
       expect('
         module Mod
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records attr attribute' do
-      expect('
-        class Klass
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records reader attribute' do
-      expect('
-        class Klass
-          attr_reader :my_attr
+          attr_writer :my_attr
         end
```

Comment:
And a test for `attr` as well, of course.

This bit is actually a little tricky, because `attr` can define setters as well, if used in the following way:

```ruby
attr :y, true
```